src/Synthesiser.cpp

```diff
@@ -141,33 +141,7 @@ static const std::string getOpContextName(const RamRelation& rel) {
     return getRelationName(rel) + "_op_ctxt";
 }
 
-class IndexMap {
-    typedef std::map<RamRelation, AutoIndex> data_t;
-    typedef typename data_t::iterator iterator;
-
-    std::map<RamRelation, AutoIndex> data;
-
-public:
-    AutoIndex& operator[](const RamRelation& rel) {
-        return data[rel];
-    }
-
-    const AutoIndex& operator[](const RamRelation& rel) const {
-        const static AutoIndex empty;
-        auto pos = data.find(rel);
-        return (pos != data.end()) ? pos->second : empty;
-    }
-
-    iterator begin() {
-        return data.begin();
-    }
-
-    iterator end() {
-        return data.end();
-    }
-};
-
-std::string getRelationType(const RamRelation& rel, std::size_t arity, const AutoIndex& indices) {
+std::string getRelationType(const RamRelation& rel, std::size_t arity, const IndexSet& indexes) {
```

Comment:
As above, except in code we use indices even more. Currently it's at 0 uses of 'indexes' and many of 'indices'.

Reply:
I agree - we need to be consistent - either we use English or American spelling in our source code. Perhaps we start a voting process for this.